package.json

```diff
@@ -21,8 +21,7 @@
     "test:coverage": "npm run build && npm test -- --no-watchman --coverage",
     "test:ci": "npm run test -- --no-watchman --maxWorkers=10",
     "test:coverage:ci": "npm run test:ci -- --coverage && codecov",
-    "release:semantic": "lerna-semantic-release pre && lerna-semantic-release post && lerna-semantic-release perform",
-    "release:manual": "./scripts/publish-manually",
+    "release": "npm run build && lerna publish --registry=https://registry.npmjs.org/ --conventional-commits",
```

Comment:
Maybe we should add `docs:publish` here?
`npm run build && lerna publish --registry=https://registry.npmjs.org/ --conventional-commits && npm run docs:publish`

Reply:
Ah fair point.

Reply:
However, I think it should only be a `postrelease` step cause it otherwise won't work as `lerna publish` triggers an interactive command.

Reply:
It should still work even if the command is interactive. As long as it exits successfully, control will then be passed to the next command (`npm run docs:publish` in this case)